Documentation/api.md

```diff
@@ -8228,6 +8228,11 @@ string
 </tr>
 </tbody>
 </table>
+<h3 id="monitoring.coreos.com/v1.PrometheusInterface">PrometheusInterface
+</h3>
+<div>
+<p>PrometheusInterface is used by Prometheus and PrometheusAgent to share common methods, e.g. config generation.</p>
+</div>
```

Comment:
I think you need to tweak https://github.com/prometheus-operator/prometheus-operator/blob/main/scripts/docs/config.json to hide the interface type from the docs.

Reply:
Do you know where can I find documentation for this tool? I couldn't find it in the original repo.

I've tried adding `PrometheusInterface` to `hideTypePatterns`, but it it made no changes to the API docs :(

Reply:
Finally got it!